go/vt/vtgate/engine/route.go

```diff
@@ -58,6 +58,9 @@ type Route struct {
 	// Query specifies the query to be executed.
 	Query string
 
+	// Table specifies the table to send the query to.
+	Table string
```

Comment:
I think I miscommunicated what I had in mind. I meant for this to be a list, and that the list of tables are accurately tracked by all primitives. However, the outermost layer exposes just the first table.
This way, we can easily extend the functionality in the future to actually display all affected tables.

But this is ok for now since we only plan to use the first table anyways. And it won't be hard to change this to include all tables when necessary.